x/ibc-rate-limit/.beaker/state.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this file needed for something?

Reply:
TBH, I'm not sure. It's part of beaker: https://github.com/osmosis-labs/osmosis/pull/2408#discussion_r946427708